pkg/logging/logging.go

```diff
@@ -34,8 +39,27 @@ var (
 	// Logger is the base logger used by Crossplane. It delegates to another
 	// logr.Logger. You *must* call SetLogger to get any actual logging.
 	Logger = logger.WithName("crossplane")
+
+	// ZapLogger is a Logger implementation.
+	ZapLogger = zapLogger
 )
 
+// ZapLogger is a Logger implementation from controller runtime.
+// If development is true, a Zap development config will be used
+// (stacktraces on warnings, no sampling), otherwise a Zap production
+// config will be used (stacktraces on errors, sampling).
+// If disableStacktrace is true, stacktraces enabled on fatals
+// independent of config.
+func zapLogger(development bool, disableStacktrace bool) logr.Logger {
+	zl := runtimezap.RawLoggerTo(os.Stderr, development)
+
+	if disableStacktrace {
```

Comment:
do you think this should use the new controller-runtime `v0.2.2` way of initializing the logger? It is probably not worth updating the entire controller-runtime to just get the new logger initialization, but it could also be nice to get a new controller-runtime update anyways.

What is your opinion @turkenh?

Reply:
As you noted, updating controller-runtime is not necessary for this specific issue but I agree that it make sense to bump controller-runtime to `v0.2.2` which is latest `v0.2`, currently it is on `v0.2.0-beta.5` which does not sound that stable :)
Still I'll go and ask on slack dev channel to make sure everyone is ok with this.